draft-ietf-quic-transport.md

```diff
@@ -428,14 +428,16 @@ A QUIC implementation SHOULD provide ways in which an application can indicate
 the relative priority of streams.  An implementation uses information provided
 by the application to determine how to allocate resources to active streams.
 
-## Required Operations on Streams {#stream-operations}
+## Operations on Streams {#stream-operations}
 
-There are certain operations that an application MUST be able to perform when
-interacting with QUIC streams.  This document does not specify an API, but
-any implementation of this version of QUIC MUST expose the ability to perform
-the operations described in this section on a QUIC stream.
+This document does not define an API for QUIC, but instead defines a set of
+functions on streams that application protocols can rely upon.  An application
+protocol can assume that an implementation of QUIC provides an interface that
```

Comment:
```suggestion
protocol can assume that a general-purpose implementation of QUIC provides an interface that
```

Reply:
I decided not to do this explicitly.  I think that it is important that application protocols are able to assume these features are present.